src/gatsby-browser.js

```diff
@@ -13,10 +14,19 @@ export const onInitialClientRender = () => {
   }
 };
 
+const getJss = customInPoint => create({
+  ...jssPreset(),
+  // Define a custom insertion point that JSS will look for when injecting the styles into the DOM.
+  insertionPoint: customInPoint ? document.getElementById(customInPoint) : undefined,
```

Comment:
The insertion point can be an HTML comment too, this approach doesn't support it.

Reply:
https://cssinjs.org/jss-api?v=v10.0.0-alpha.16#setup-jss-instance